src/python/dbs/apis/dbsClient.py

```diff
@@ -400,7 +411,17 @@ def __callServer(self, method="", params={}, data={}, callmethod='GET', content=
                 print("HTTP={} URL={} method={} params={} data={} headers={}".format(callmethod, self.url, method, params, data, request_headers))
             self.http_response = method_func(self.url, method, params, data, request_headers)
         except HTTPError as http_error:
-            self.__parseForException(http_error)
+            if headerHas(http_error.header, 'Content-Type', 'application/json'):
```

Comment:
Shouldn't it support `application/ndjson` as well?

Reply:
No, the `application/ndjson` will not work here since the code uses `json.loads`, as such the application/ndjson is a string of `{some-json}\n{another-json}\n` and it is not valid JSON, e.g.
```
>>> d="""{"foo":1}\n{"foo":2}"""
>>> import json
>>> json.loads(d)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/decoder.py", line 367, in decode
    raise ValueError(errmsg("Extra data", s, end, len(s)))
ValueError: Extra data: line 2 column 1 - line 2 column 10 (char 10 - 19)
```